core/src/main/java/com/scalar/db/storage/dynamo/StatementHandler.java

```diff
@@ -1,60 +0,0 @@
-package com.scalar.db.storage.dynamo;
```

Comment:
I deleted `StatementHandler` in this PR because I needed to change the return type of `handle()`.